src/libav_streamer.cpp

```diff
@@ -77,7 +77,7 @@ LibavStreamer::~LibavStreamer()
 }
 
 // output callback for ffmpeg IO context
-static int dispatch_output_packet(void * opaque, uint8_t * buffer, int buffer_size)
+static int dispatch_output_packet(void * opaque, const uint8_t * buffer, int buffer_size)
```

Comment:
The old signatures have been deprecated in libavformat version 61. We can check for the version in the C preprocessor stage.
```suggestion
#if LIBAVFORMAT_VERSION_MAJOR < 61
static int dispatch_output_packet(void * opaque, uint8_t * buffer, int buffer_size)
#else
static int dispatch_output_packet(void * opaque, const uint8_t * buffer, int buffer_size)
#endif
```